apps/cowswap-frontend/src/legacy/state/orders/updaters/UnfillableOrdersUpdater.ts

```diff
@@ -150,6 +150,9 @@ export function UnfillableOrdersUpdater(): null {
           account,
           amount: currencyAmount,
           balances: balancesRef.current,
+          spender: undefined, // Only the balance is relevant for determining if we should use verified quotes
```

Comment:
This comment does not match the property. At least seems like a copy&past error.

Reply:
can look confusing, but it just tried to say that allowances ar enot needed, and this is why the spender is undefined. 

If you want to check also the allowances, you need to pass both `spender` and `allowances`

Reply:
I will try to revampt this PR into another PR, it has fallen out of sync